ote/src/cljs/ote/views/transport_service_common.cljs

```diff
@@ -170,7 +170,7 @@
                        :tooltip (tr [:form-help :external-interfaces-tooltips :external-service-url])
                        :width "20%"
                        :full-width? true
-                       :on-blur #(e! (ts/->EnsureExternalInterfaceUrl (-> % .-target .-value)))
+                       :on-blur #(e! (ts/->EnsureExternalInterfaceUrl (-> % .-target .-value) "GTFS")) ;; TODO: Get format from state
```

Comment:
Is this hardcoded gtfs value ok for now?

Reply:
Nope, still working on it.